symfony/test/functional/cypress/fixtures/pim-termination-reasons-config.json

```diff
@@ -0,0 +1,67 @@
+{
+    "TC01":{
```

Comment:
The reason for this approach is, we can separate the data which utilized by each test method ("it"). I thought this will add an advantage in a situation like, when have to do modifications only on a specific test method. So, anyone can easily locate the particular data which only corresponds to the test method and do the modification in the future.

I was meant like this - TC01 = Test Case 01.
Since the exact name that I have used within the corresponding test case (within "it") is lengthy.
Shall I rename it like TestCase01?

Reply:
It's great to use `PageObject` model and `dataseperation` but that variable not make any sense and this file contains more test data that not relevant each other